components/Header.tsx

```
@@ -29,14 +29,23 @@ const Header: React.FC<HeaderProps> = ({ pageTitle, altColor, altBack }) => {
                     backgroundColor: altBack ? "#f72f2f" : "#fff"
                 }}
             >
-                <DrawerButton iconColor={altColor ? "#fff" : primaryColor}/>
+                <DrawerButton iconColor={altColor ? "#fff" : primaryColor} />
 
-                {pageTitle === 'Order History' || 'Settings' ? <Text style= {{fontSize: 20, fontWeight: 600, color: pageTitle ===  'Order History' ? '#000' : "#fff"}}>{pageTitle}</Text> : null }
+                {pageTitle === 'Order History' || 'Settings' ? <Text style={{ fontSize: 20, fontWeight: 600, color: pageTitle === 'Order History' ? '#000' : "#fff" }}>{pageTitle}</Text> : null}
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)